doc/api/HostDB.md

```diff
@@ -288,3 +402,46 @@ Examples
   ]
 }
 ```
+
+#### Hosts
+
+###### Request
+```
+/hostdb/hosts/ed25519:1234567890abcdef1234567890abcdef1234567890abcdef1234567890abcdef
```

Comment:
should we return the string value of the public key from the API? I'm guessing that we should. We should probably add it as an extra field to the entry.